.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- This is a template - do add to or remove from as needed -->
+<!-- Please provide a general summary of your changes in the Title above 🚀 -->
+
+## Description 💬
+<!-- Please describe your changes in detail -->
+
+## How Has This Been Tested? 🖥️
+[Preview](https://<branch>--splitify-preview.netlify.app)
+
```

Comment:
Netlify will automatically build and post a link to the preview of the current PR. I reckon branch deploy previews aren't critical to include as part of the PR
```suggestion
```

Reply:
Where does it post the preview link?
I'd kindof like it to be here so that it's really accessible when you're asked to review

Reply:
@CarlinWilliamson  inside the merge checks, there is a check: `netlify/splitify-preview/deploy-preview — Deploy preview ready! ` - clicking on that will link to the build preview

Reply:
Oh sweet. 10/10